### PR TITLE
new

### PR DESCRIPTION
**Pull request by AbenezerAssefa**
adding the about, comment me adding CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,6 +54,230 @@ <h1>
...
+                    <hr>
+                </div>
+
+                <div class="skills">
+                    <p>Skills</p>
+                    <img class="arrow-img" src="./img/chevron-down.svg" alt="Click to see the side bar">
+                    <hr>
+                </div>
+
+            </div>
```
Comment (didierganthier) [inline index.html:250]: Well done here 👏🏽 great project but the technology categories ("Languages", "Frameworks" and "Skills") should in a `ul`, each of them in a `li` for better semantics. 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @AbenezerAssefa ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Correct GitHub flow ✔️ 
- Linters passing ✔️ 
- Descriptive PR ✔️ 

## Required Changes 🛠
- [ ] To make the repo more professional and cool 😎 make sure to use the latest README template provided by Microverse. It can be found [here](https://github.com/microverseinc/readme-template/blob/master/README.md)

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: index.html (hunk 1 above)
Comment (divinecharlotte) [inline index.html:250]: - [ ] kindly consider implementing the addressed issues here as requested by the previous reviewer for better semantics. 😉

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @AbenezerAssefa 👋, 

## STATUS: REQUESTED CHANGES ♻️ 

Good job so far 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!


### To Highlight 🎯 💡 


## Required Changes ♻️



_Check the comments under the review._


## Optional suggestions

- [ ] Kindly consider checking back your Readme to make it contain information about your project rather than copy-pasting the readme which is not professional

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**